docs/learn/dapp-developer/transfer-lyx.md

```diff
@@ -19,10 +19,17 @@ import TabItem from '@theme/TabItem';
 <br /><br />
 </div>
 
-In this guide, we will learn **how to transfer LYX** from your Universal Profile to any `address` (including another Universal Profile :up: ). We will cover:
+:::tip Generic Transaction Handling
 
-- `sendTransaction({from:, to, value, data: '0x', ....})`
-- Give example with Web3js v1 / Ethers -> <https://www.quicknode.com/guides/ethereum-development/transactions/how-to-send-a-transaction-in-ethersjs>
+This guide can be used to **transfer LYX** from an Universal Profile to any `address`, including another profiles or vaults. As LUKSO is EVM-compatible, all regular Ethereum providers can be used with their regular `.sendTransaction()` functions.
+
+:::
+
+:::caution Library Compatability
+
+The ⚒️ [erc725.js](https://docs.lukso.tech/tools/erc725js/getting-started) library only works up to `web3` Version `1.0.0`. If you are planning to using it for reading profile data, we recommend downgrading `web3` or using other libraries like `ethers` or `ethereum`.
+
```

Comment:
What is the reason for mentioning a erc725.js web3 compatibility if you don't use erc725.js in this toturial?